bracketcolors/README

```diff
@@ -0,0 +1,32 @@
+Color brackets, parenthesis, and braces
+===================
```

Comment:
RestructuredText (the text format used here) requires that underlines are exactly as long as the heading above.

Reply:
thanks, fixed!